packages/xo-server/config.toml

```diff
@@ -48,6 +48,7 @@ maxTokenValidity = '0.5 year'
 [backup]
 # Delay for which backups listing on a remote is cached
 listingDebounce = '1 min'
+vmBackupSizeTimeout = '2 seconds'
```

Comment:
Would be nice to add a little comment explaining how/when this value is used and why it needs to be small?

Reply:
I have improved it but I am not a commentary specialist :p So if you have a better idea :)

Reply:
Yep, I changed it :win